queries/puppet/indents.scm

```diff
@@ -0,0 +1,31 @@
+[
+  (block)
+  (defined_resource_type)
+  (parameter_list)
+  (attribute_type_entry)
+  (resource_declaration)
+  (selector)
+  (method_call)
+  (case_statement)
+  (hash)
+  (array)
+] @indent.begin
+
+
+[
+  "}"
+  "]"
+  ")"
+] @indent.end
+
+[ "{" "}" ] @indent.branch
+
+[ "[" "]" ] @indent.branch
+
+[ "(" ")" ] @indent.branch
```

Comment:
I'd suggest not setting opening brackets to `@indent.branch`, unless all of its parent's nodes are marked with `@indent.begin`

Reply:
so just closed ones? or do you mean just for [ ]

Reply:
Just closed ones. Opening ones are messier to handle correctly